app/src/main/java/com/example/android/service/BackendClient.java

```diff
@@ -1,16 +1,23 @@
 package com.example.android.service;
 
+import org.json.JSONObject;
+
 import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Objects;
 
+import okhttp3.MediaType;
 import okhttp3.OkHttpClient;
 import okhttp3.Request;
+import okhttp3.RequestBody;
 import okhttp3.Response;
 
 public class BackendClient {
     private static BackendClient instance = null;
     private final OkHttpClient _client;
-    private final String _baseUrl = "https://fa43-219-85-87-19.ap.ngrok.io";
+    private final String _baseUrl = "https://2f43-2001-b011-4002-332b-1074-bb39-423b-aa5f.jp.ngrok.io";
+    MediaType JSON = MediaType.parse("application/json; charset=utf-8");
```

Comment:
Suggest making it `private` and `final`. 
Since it has special usage, the current variable naming style is acceptable in my opinion. 

```suggestion
    private final MediaType JSON = MediaType.parse("application/json; charset=utf-8");
```